### PR TITLE
Hwg170030

### PR DESCRIPTION
**Pull request by nith822**
My implementation of adding a preview violates so many design principles

### CODE REVIEW

File: UI/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ const routes: Routes = [
 	{ 
 		path: 'login', 
 		loadChildren: () => import('./main/modules/login/login.module').then(m => m.LoginModule) 
-	}
+	},
```
Comment (mattAbe) [inline UI/src/app/app-routing.module.ts:17]: will this ',' create a syntax error?

File: UI/src/app/main/modules/upload/upload.module.ts [hunk 2]
```diff
@@ -3,18 +3,25 @@ import { CommonModule } from '@angular/common';
...
 
 import { UploadComponent } from './upload.component';
 import { UploadService } from '../../upload/upload.service';
 import { UploadPreviewComponent } from './upload-preview/upload-preview.component';
 
 
+ 
 @NgModule({
-  declarations: [UploadComponent, UploadPreviewComponent],
+  declarations: [UploadComponent, UploadPreviewComponent,],
```
Comment (mattAbe) [inline UI/src/app/main/modules/upload/upload.module.ts:17]: is this ',' a syntax error during runtime?

File: UI/src/app/main/upload/upload.service.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ export class UploadService {
 	
 	tags: string;
 	
-	constructor(private http: HttpClient, private userService: UserService, private router: Router) { 
+	constructor(private http: HttpClient, private userService: UserService, private router: Router,) {
```
Comment (mattAbe) [inline UI/src/app/main/upload/upload.service.ts:32]: same ','

**Comment by sxb152930** [comment]
LBTM

**Pull request merged**